launch/tier4_control_launch/config/trajectory_follower/longitudinal_controller.param.yaml

```diff
@@ -1,11 +1,11 @@
 /**:
   ros__parameters:
-    longitudinal_ctrl_period: 0.03  # control period [s]
     delay_compensation_time: 0.17
 
     enable_smooth_stop: true
     enable_overshoot_emergency: true
     enable_slope_compensation: false
+    enable_keep_stopped_until_steer_convergence: true
```

Comment:
better to set false by default

Reply:
[b51a021](https://github.com/autowarefoundation/autoware.universe/pull/901/commits/b51a021429fcd6bd41642c52866f154b4797bdc6)